exercises/practice/robot-name/.approaches/introduction.md

```diff
@@ -116,9 +115,9 @@ it is often useful to consider the tradeoffs of different ways of representing i
 
 Converting names to integers and back again might seem tricky at first.
 A decent intuition is that the names are a number, but in an inconsistent base.
-Just as 153 is 3 + 5 * 10 + 1 * 100,
+Just as 153 is 3 + 5 _ 10 + 1 _ 100,
```

Comment:
```suggestion
Just as 153 is 3 + 5 * 10 + 1 * 100,
```

The *s here are for multiplication, not emphasis. They don't need to be escaped if they're surrounded by whitespace.